includes/admin/views/html-notice-review.php

```diff
@@ -13,17 +13,19 @@
 		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18.15,4l1.23,2H15.49L14.26,4ZM20,20H2.21L12,4.09,18.1,14H10.77L12,12h2.52L12,7.91,5.79,18H20.56l1.23,2ZM17.94,10,16.71,8H20.6l1.23,2Z"/></svg>
 	</div>
 	<div class="everest-forms-message--content">
-		<h3 class="everest-forms-message__title"><?php esc_html_e( 'Please help us spread the word', 'everest-forms' ); ?></h3>
+		<h3 class="everest-forms-message__title"><?php esc_html_e( 'HAKUNA MATATA!', 'everest-forms' ); ?></h3>
```

Comment:
@ujw0l Was this intentional seriously "HAKUNA MATATA!"?

Reply:
@shivapoudel yes this was intentional